golem-worker-executor-base/src/storage/blob/s3.rs

```diff
@@ -283,11 +282,11 @@ impl BlobStorage for S3BlobStorage {
     ) -> Result<Option<Bytes>, String> {
         let bucket = self.bucket_of(&namespace);
         let key = self.prefix_of(&namespace).join(path);
-        let description = format!("Downloading blob from {bucket}::{key:?}");
+
         let result = with_retries(
-            &description,
             target_label,
             op_label,
+            Some(format!("{} - {:?}", bucket, key)),
```

Comment:
fyi: these format string can be inlined which in my opinion is more readable (but probably it is a matter of taste :) (like `format!("{bucket} - {key:?}")`)

Reply:
https://github.com/golemcloud/golem/pull/620/commits/d2a9f1e46a432d0293aa4cbaf123d840b9278d76